packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotMessage/BotMessage.tsx

```diff
@@ -61,6 +61,13 @@ export default MessageLoading;
       <Message name="Bot" role="bot" avatar={patternflyAvatar} content={orderedList} />
```

Comment:
Maybe we can change "Example content with updated timestamp text" from the first bot message to be "Text-based message from a bot named "Bot", with updated timestamp"? Maybe sounds a little weird, but it could be good to point out that there's already a name with these messages, I tbh didn't notice myself in previous prs.

Longer term, I wonder if we could rename it to "FlyBot" or something, just to make it more "real" and obvious. Just a side comment/low priority thing

Reply:
I just pushed these changes. We did have this issue come up: https://github.com/patternfly/virtual-assistant/issues/299. I think we do need improved docs around timestamps. I added a line about it, but let me know if you envision something else. I figure we can update the Message docs here as part of that, but I'll update our demo examples to include timestamps before I close #299.

Reply:
There was a missing quotation mark, but otherwise everything looks good to me! That timestamps plan sounds good to me